src/main/java/Edit_Group_Profile_Use_Case/EditGroupProfileRequestModel.java

```diff
@@ -0,0 +1,108 @@
+package Edit_Group_Profile_Use_Case;
+import java.util.HashMap;
+import java.util.*;
+
+public class EditGroupProfileRequestModel {
+    private String groupName;
+    private String courseCode;
+    private String description;
+    private String timeCommitment; //make sure not negative number
+    private String location;
+    private String meetingTime; //morning, afternoon, evening or day of the week
+
+    public EditGroupProfileRequestModel(String groupName, String description, String timeCommitment, String location, String meetingTime, String courseCode) {
+        this.groupName = groupName;
+        this.description = description;
+        this.timeCommitment = timeCommitment;
+        this.location = location;
+        this.meetingTime = meetingTime;
+        this.courseCode = courseCode;
+    }
+
+
+    public String getGroupName() {
+        return this.groupName;
+    }
+
+    public void setGroupName(String groupName) {
+        this.groupName = groupName;
+    }
+
```

Comment:
In my implementation of the Group Entity, I am making group names unique so we can use them as an identifier for groups. Will group names be able to be changed? I think we should discuss this decision